src/builder.py

```diff
@@ -213,6 +213,10 @@ def run(config: state.BuilderRunConfig) -> None:
                     ">>",
                     str(OUTPUT_LOG_PATH),
                     "2>&1",
+                    "||",
+                    # Run the callback script without Openstack ID argument to let the charm know
+                    # about the error.
+                    str(config.callback_script),
```

Comment:
Do you think this should be a different callback event rather than implicitly indicating failure by not passing a value?